AAI/AAIConnectProfile.md

```diff
@@ -98,6 +112,15 @@ described herein, encoded via JWS Compact Serialization per
 [RFC7515](https://datatracker.ietf.org/doc/html/rfc7515), containing
 `ga4gh_passport_v1` as a space-separated entry within the `scope` claim but
 does not contain [GA4GH Claims](#term-ga4gh-claim).
+<a name="term-embedded-token-issuer"></a> **Embedded Token Issuer** --
```

Comment:
From Security Subgroup call: Let's change this to Visa Issuer. At one time there was an expecation that there might be other types of embedded tokens, but currently there are only Visas. We should stay concrete and generalize later if needed.

Reply:
Updated.